tests/device_tests/cardano/test_address_public_key.py

```diff
@@ -0,0 +1,49 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib.cardano import get_address, get_public_key
+from trezorlib.tools import parse_path
+
+from ...common import parametrize_using_common_fixtures
+
+pytestmark = [
+    pytest.mark.altcoin,
+    pytest.mark.cardano,
+    pytest.mark.skip_t1,
```

Comment:
seems that you could add `skip_ui` here too?

Reply:
ac9f57fa8e1406efa6360fdb53ace5962940be1e